src/plots/gl2d/scene2d.js

```diff
@@ -522,6 +522,21 @@ proto.draw = function() {
                             glplot.pixelRatio
                 ];
 
+                // this needs to happen before the next block that deletes traceCoord data
+                // also it's important to copy, otherwise data is lost by the time event data is read
+                this.graphDiv.emit('plotly_hover', {
+                    points: [{
+                        trace: nextSelection.trace,
+                        dataCoord: nextSelection.dataCoord.slice(),
+                        traceCoord: nextSelection.traceCoord.slice(),
+                        textLabel: nextSelection.textLabel,
+                        color: nextSelection.color,
+                        name: nextSelection.name,
+                        hoverinfo: nextSelection.hoverinfo,
+                        screenCoord: nextSelection.screenCoord.slice()
```

Comment:
.. PS. if you meant to ask if I have all the details in `scene2d.js` to raise a `plotly_hover` event in identical structure with the SVG counterparts, then the answer is, I'd need to cut and paste so much of what's in `fx.hover` already that I'd say no :-) i.e. doable and basic inputs are available but would lead to lots of duplication between `scene2d.js` and `graph_interact.js`.